ocrd_eval.schema.yml

```diff
@@ -0,0 +1,192 @@
+      - eval_workflow
+      - eval_workspace
+      - gt_workspace
+      - document_metadata
+
+    properties:
+
+      ocr_workflow:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The OCR-D workflow that produced the ocr_workspace
+
+      ocr_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the OCR
+
+      eval_workflow:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The OCR-D workflow that produced the eval_workspace
+
+      eval_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the evaluation results
+
+      gt_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the GT
+
+      workflow_steps:
+        type: object
+        description: Human readable description of the individual steps in the workflow (for UI)
+        patternProperties:
+          '^[0-9]+$':
+            type: string
+            description: Description of this workflow step
+
+      workflow_model:
+        type: string
+        description: Human readable name of the main model used for recognition in the OCR workflow (for UI)
+
+      eval_tool:
+        type: string
+        description: Human readable name and version of evaluation tool used (for UI
+
+      document_metadata:
+        type: object
+        title: Bibliographical and typographical metadata about the work to be evaluated
+        properties:
+
+          publication_year:
+            type: number
+            description: Year he document was originally published
+
+          publication_century:
+            type: string
+            description: Century he document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
+
+          publication_decade:
+            type: string
+            description: Decade the document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
+
+          number_of_pages:
+            type: number
+            description: Number of pages in this work (i.e. the number of images in the gt_workspace)
+
+          layout:
+            type: string
+            enum: ['simple', 'complex']
+
+          fonts:
+            type: array
+            items:
+              type: string
+              enum: ['antiqua', 'fraktur']
+
+      provenance:
+        type: object
+        description: Information on which tools in which version were used in determining metrics
+        properties:
+          paramters:
+            type: object
+            description: Parameters passed to the evaluation processor
+
+  EvaluationReport:
+    type: object
+    additionalProperties: false
+    description: The metrics measured for this document
+    properties:
+      document_wide:
+        type: object
+        description: Document-wide metrics
+        #properties: { $ref: '#$defs/EvaluationMetrics' }
```

Comment:
I'd rather have no lines in comments for the first draft. When will we need this?